### DISCUSSION TITLE
Loki: Add search for labels as implemented for values

### DISCUSSION THREAD

**Discussion by MB175** [grafana/grafana#48460] [Data Sources]
Adding a search for Labels in the Loki explorer as it already is implemented for values would be a great improvement in the workflow.

https://github.com/grafana/grafana/blob/277a83215c1e20ff8c93768f93edf92d60deee5f/public/app/plugins/datasource/loki/components/LokiLabelBrowser.tsx#L454

**Comment by mellieA** [comment]
@MB175 I've moved your issue into discussions, this is the new home for feature requests. 

You can read more about that change here: https://github.com/grafana/grafana/discussions/36962

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
